src/fsharp/Interactive.DependencyManager/foo.sig

```diff
@@ -0,0 +1,188 @@
+#light
```

Comment:
(Putting this at the top)

What is the significance of the term `foo` for the name of this file?